scripts/src/java/org/oppia/android/scripts/common/ScriptResultConstants.kt

```diff
@@ -4,3 +4,7 @@ package org.oppia.android.scripts.common
 const val REGEX_CHECK_PASSED_OUTPUT_INDICATOR: String = "REGEX PATTERN CHECKS PASSED"
 /** RegexPatternValidationCheck failed indicator. */
 const val REGEX_CHECK_FAILED_OUTPUT_INDICATOR: String = "REGEX PATTERN CHECKS FAILED"
+/** XmlSyntaxCheck successfuly passed indicator. */
```

Comment:
Nit: please add spaces between the constants such that it looks like this:

```kotlin
/** KDoc...*/
constant

/** KDoc... */
constant

...
```

Reply:
Done.